apps/omg_api/test/api/core_test.exs

```diff
@@ -99,68 +93,37 @@ defmodule OMG.API.CoreTest do
 
   @tag fixtures: [:alice]
   test "transaction must have distinct inputs", %{alice: alice} do
-    duplicate_inputs = TestHelper.create_encoded([{1, 2, 3, alice}, {1, 2, 3, alice}], eth(), [{alice, 7}])
+    duplicate_inputs = TestHelper.create_encoded([alice], [{1, 2, 3}, {1, 2, 3}], [{alice, eth(), 7}])
 
     assert {:error, :duplicate_inputs} = Core.recover_tx(duplicate_inputs)
   end
 
-  @tag fixtures: [:alice, :bob]
-  test "transaction is not allowed to have input and empty sig", %{alice: alice, bob: bob} do
-    full_signed_tx = TestHelper.create_signed([{1, 2, 3, alice}, {2, 3, 4, bob}], eth(), [{alice, 7}])
-
-    missing1 =
-      %Transaction.Signed{full_signed_tx | sig1: @empty_signature}
-      |> Transaction.Signed.encode()
-
-    missing2 =
-      %Transaction.Signed{full_signed_tx | sig2: @empty_signature}
-      |> Transaction.Signed.encode()
-
-    partial_signed_tx1 = TestHelper.create_signed([{1, 2, 3, alice}], eth(), [{alice, 7}])
-
-    missing3 =
-      %Transaction.Signed{partial_signed_tx1 | sig1: @empty_signature}
-      |> Transaction.Signed.encode()
-
-    partial_signed_tx2 = TestHelper.create_signed([{0, 0, 0, %{priv: <<>>}}, {1, 2, 3, alice}], eth(), [{alice, 7}])
-
-    missing4 =
-      %Transaction.Signed{partial_signed_tx2 | sig2: @empty_signature}
-      |> Transaction.Signed.encode()
-
-    assert {:error, :signature_missing_for_input} == Core.recover_tx(missing1)
-    assert {:error, :signature_missing_for_input} == Core.recover_tx(missing2)
-    assert {:error, :signature_missing_for_input} == Core.recover_tx(missing3)
-    assert {:error, :signature_missing_for_input} == Core.recover_tx(missing4)
+  @tag fixtures: [:alice]
+  test "transaction is not allowed to have input and empty sigs", %{alice: alice} do
```

Comment:
this test should be reverted to follow the old flow - create a valid tx and then take the sigs away. No need for the new API of `create_signed`